frameworks/FSharp/giraffe/src/App/Program.fs

```diff
@@ -0,0 +1,62 @@
+module App.App
+
+open Microsoft.AspNetCore.Hosting
+open Giraffe
+
+[<CLIMutable>] 
+type JsonMessage = { message : string }
+
+[<CLIMutable>][<Struct>] 
+type JsonStructMessage = { message : string }
+
+type Implementation = Stock | Custom
+
+module Routes =
+
+    let stock : HttpHandler list = 
+        [ route "/plaintext" >=> text "Hello, World!"
+          route "/json" >=> json { JsonMessage.message = "Hello, World!" } ]
+
+    let custom : HttpHandler list = 
+        let inline contentLength (x:int32) = new System.Nullable<int64>( int64 x )
+
+        let json data : HttpHandler =

```

Comment:
I think we should use the giraffe helpers here.

Reply:
Helpers are already used in stock? (idiomatic) 

Static conversions are used for custom plaintext, and utf8json lib (benchmark optimized)

Both varieties will show up in the benchmarks

Reply:
Ah didn't see that. Thanks